common/helpers/move/get-assessments.js

```diff
@@ -0,0 +1,80 @@
+const { find, map, sortBy } = require('lodash')
```

Comment:
I know this is a moved from elsewhere, but there is a lot going on in this file that exports a single function that needs to all be paid attention to to understand what's going on. And the tests for this were so large that Github didn't automatically show them because they're too big.

Possibly not the time for this, but I wonder if there's a way of simplifying this?

Reply:
Absolutely. But I'm going to punt that to Dom to take further - I'm unclear as to which of these bits are independent or really depend on each other and whether any of the bits ever get used independently